react/modules/tasks/nextjs-ssr-ssg.md

```diff
@@ -0,0 +1,41 @@
+# Next.js. Server Side Rendering
+
+## Technical requirements
+
+1. Create a separate branch for this task from the previous branch task. Branch name: "nextjs-ssr-pages-api".
+2. Next.js Pages API Integration:
```

Comment:
I guess it is worth mentioning that `react-router` should be removed in NextJS part and routing SHOULD be based on the NextJS, otherwise there will be students which will complain that it wasn't stated explicitly.